salt/modules/nilrt_ip.py

```diff
@@ -9,30 +9,42 @@
 import logging
 import time
 import os
+import re
 
 # Import salt libs
 import salt.exceptions
 import salt.utils.files
 import salt.utils.validate.net
 
 # Import 3rd-party libs
+# pylint: disable=import-error,redefined-builtin,no-name-in-module
+from salt.ext.six.moves import map, range
```

Comment:
so, you don't need a try except loop for configparser, it just needs to be added to the imports on this line.